apps/native/app/src/ui/lib/empty-state/empty-list.tsx

```diff
@@ -48,3 +69,13 @@ export function EmptyList({ title, description, image }: HeadingProps) {
     </Host>
   )
 }
+
+export function EmptyListSmall({ title, description, image }: HeadingProps) {
```

Comment:
I would rather use size property here to the EmptyList and that component would determine if the list should be small or not